test/sql/ordinality/a0.csv

```diff
@@ -0,0 +1,11 @@
+Numbers, Chars
```

Comment:
Instead of adding all of these data files - can we use existing data files or generate data in the tests themselves?